src/pybind/mgr/dashboard/controllers/cephfs.py

```diff
@@ -468,7 +468,7 @@ def get(self):
 
 @UiApiController('/cephfs', Scope.CEPHFS)
 class CephFsUi(CephFS):
-    RESOURCE_ID = 'fs_id'
+    RESOURCE_ID = 'fs_id'  # type: ignore
```

Comment:
```suggestion
    RESOURCE_ID = 'fs_id'  # type: str
```
?

Reply:
Doesn't work, have tried this already.
```
dashboard/plugins/feature_toggles.py:16: note: In module imported here,
dashboard/module.py:56: note: ... from here:
dashboard/controllers/cephfs.py: note: In class "CephFsUi":
dashboard/controllers/cephfs.py:471: error: Incompatible types in assignment (expression has type "str", base class "RESTController" defined the type as "None")
```

Reply:
you'll need to annotate 

https://github.com/ceph/ceph/blob/83132bff7e19b2ddd920900377e070150bbc7dd6/src/pybind/mgr/dashboard/controllers/__init__.py#L736

with 

```python
# type: Optional[str]
```

Reply:
Fixed